server/chat-plugins/chat-monitor.ts

```diff
@@ -320,55 +320,58 @@ void FS(MONITOR_FILE).readIfExists().then(data => {
 
 /* The sucrase transformation of optional chaining is too expensive to be used in a hot function like this. */
 /* eslint-disable @typescript-eslint/prefer-optional-chain */
-export const chatfilter: ChatFilter = function (message, user, room) {
```

Comment:
One thing that would be interesting here (and also not change indentation level) would be:

```ts
export const chatfilter: ChatFilter = function (message, user, room) {
  ...
};
chatfilter.priority = 0;
```

Which can be left off for most chatfilters at default priority.

Reply:
I would leave it up to you, but I would definitely strongly prefer that approach.

Reply:
That's definitely simpler.

Reply:
I'll change to that